pages/eligibility.tsx

```diff
@@ -103,4 +112,21 @@ const EligibilityForm: React.FC<EligibilityFormProps> = ({
   </MarginCard>
 )
 
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const {projectId} = context.query
```

Comment:
I couldn't find a way to use sessionStorage in getServerSideProps, so I am also utilizing projectId in both the eligibility and study-consent pages.